src/components/DateEntry/DateEntry.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import DateEntry from './DateEntry'
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {
+  title: 'Capstone Project/4-Meals',
+  component: DateEntry,
+}
+
+const Template = args => (
+  <DateEntry
+    {...args}
+    date={'HEUTE'}
+    mealList={{
+      breakfast: 'Smoothie',
+      lunch: 'Salmon wrap',
+      dinner: 'Scrambled eggs',
+      snack: 'Apple',
+    }}
+  />
+)
+
+export const MealDateNavigation = Template.bind({})
```

Comment:
You need to test all props that you currently have for your `DateEntry` component. Here is how I'd implement it. If you have any questions, please ask away.

```suggestion
import { action } from '@storybook/addon-actions'

export default {
  title: 'Capstone Project/Meals',
  component: DateEntry,
}

const Template = args => <DateEntry {...args} />

export const Primary = Template.bind({})

Primary.args = {
date: 2021-03-31,
mealList: { 
  breakfast: 'Smoothie',
  lunch: 'Salmon wrap',
  dinner: 'Scrambled eggs',
  snack: 'Apple'
 },
 daily: 2,
 onPreviousDay: action('onPreviousDay'),
 onNextDay: action('onNextDay')
}
```